### PR TITLE
feat: add a version command

### LINKED ISSUE #127: Add a version command
**Is your feature request related to a problem? Please describe.**
As I was filling the report for #126 there's an ask for version but no easy way to find the version of the bin I have installed, I have to go and do `brew info oasdiff`

**Describe the solution you'd like**
Add a `oasdiff version` or `oasdif -version` option to the CLI, given the use of go native flag implementation more inclined to the use of `oasdiff version` to stay consistent with other go tooling

**Describe alternatives you've considered**
`brew info oasdiff` only works when installed via brew and is not a comamnd many will be familiar vs the common `version` option of many CLIs

**Additional context**
Happy to help to implement this feature as I've done it before for other go CLIs as long as we agree on the option

### PR DESCRIPTION
**Pull request by gssbzn**
closes #127

Output for `go build`
```bash
./oasdiff version
oasdiff version: main
```

Output for `docker build .`
```bash
docker run oasdiff version
oasdiff version: v1.1.39-5-ga22039b
```

Output for `goreleaser build --snapshot`
```bash
./oasdiff version
oasdiff version: 1.1.39-SNAPSHOT-a22039b
```

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ builds:
      - arm64
    env:
      - CGO_ENABLED=0
+   ldflags:
+     - -s -w -X github.com/tufin/oasdiff/build.Version={{.Version}}
```
Comment (reuvenharrison) [inline .goreleaser.yml:13]: Isn't it sufficient to use:
- -s -w -X build.Version={{.Version}}

This way we decouple from the repo location and name.

File: main.go [hunk 2]
```diff
@@ -39,6 +40,7 @@ func init() {
 	flag.BoolVar(&breakingOnly, "breaking-only", false, "display breaking changes only")
 	flag.StringVar(&format, "format", formatYAML, "output format: yaml, text or html")
 	flag.BoolVar(&failOnDiff, "fail-on-diff", false, "fail with exit code 1 if a difference is found")
+	flag.BoolVar(&version, "version", false, "show version and quit")
```
Comment (reuvenharrison) [inline main.go:43]: Doing it this way standardizes the ```version``` flag so it behaves like all other flags.

File: .goreleaser.yml (hunk 1 above)
Comment (gssbzn) [reply .goreleaser.yml:13]: not when using go modules, it depends on this line https://github.com/Tufin/oasdiff/blob/a39d70503d88953d2f4c0c2886b3cb9ffce0e263/go.mod#L1

This is also why it works for my fork

File: main.go (hunk 2 above)
Comment (gssbzn) [reply main.go:43]: works for me

**Pull request merged**